src/test/app/claims/models/claim.ts

```diff
@@ -1044,6 +1044,19 @@ describe('Claim', () => {
     })
   })
 
+  describe('isTransferred', () => {
+    let claim
+
+    beforeEach(() => {
+      claim = new Claim()
+      claim.state = 'TRANSFERRED'
+      claim.responseDeadline = MomentFactory.currentDate()
```

Comment:
I don't think we need this line

Reply:
This is needed as some of the calculations to determine what the status is depend on a value in this variable